docs/user/04-metrics.md

```diff
@@ -860,3 +860,32 @@ spec:
 **Cause**: Gateway cannot receive metrics at the given rate.
 
 **Solution**: Manually scale out the gateway by increasing the number of replicas for the metric gateway. See [Module Configuration and Status](https://kyma-project.io/#/telemetry-manager/user/01-manager?id=module-configuration).
+
+### Scrape metrics from system namespaces
+
+**Symptom**: Metrics from system namespaces are not scraped
+
+**Cause**: By default metrics from system namespaces are not scraped.
```

Comment:
```suggestion
**Cause**: This is the default behaviour if you have defined a namespace selector. For details, see [9. Add Filters](#_9-add-filters).
```
Did I understand this correctly? I assume that's because of the filter settings, right?
> The default settings depend on the input:
> If no namespace selector is defined for the prometheus or runtime input, then metrics from system namespaces are excluded by default.
> However, if the namespace selector is not defined for the istio and otlp input, then metrics from system namespaces are included by default.

Also, pls check if the suggested link works.